policies/adding-new-maintainers.md

```diff
@@ -0,0 +1,23 @@
+This document describes the process for adding new people to maintainer roles.
+"Maintainer" here basically means any role with github write powers. The
+process is in the hands of the Coordination Committee according to
+[APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst), which also
+requires the comment period. The process is:
```

Comment:
I don't see anything in APE0 about maintainers, so it's not clear to me what the "which" in "which also requires the comment period" refers to.

Reply:
The APE0 part is about *all* the roles, not just maintainer roles.  But agreed this is confusingly worded, so I'll accept adrian's suggestion making this unnecessary!